src/components/header/header.component.jsx

```diff
@@ -17,14 +18,9 @@ export const Header = ({ routes, onOpenMobileMenu }) => {
           <ul className={classes.linkList}>
             {routes.map((route, index) => (
               <li key={index} className={classes.linkWrapper}>
-                <Link
-                  className={classes.link}
-                  to={route.to}
-                >
-                  {route.label === 'Hermez' ? (
-                    <Logo />
-                  ) : route.label}
-                </Link>
+                {route.id === 1
```

Comment:
same with the id here

Reply:
Those are being passed from layout component. Comment added.